hi-ml/testhiml/testhiml/test_attentionlayers.py

```diff
@@ -23,6 +23,8 @@ def _test_attention_layer(attentionlayer: nn.Module, dim_in: int, dim_att: int,
         pass
     elif isinstance(attentionlayer, MaxPoolingLayer):
         pass
+    elif isinstance(attentionlayer, TransformerPoolingBenchmark):
```

Comment:
[Minor] To avoid having too many cases here, we can instead use a single `isinstance()` with a tuple of all the types we want to check at once:
```python
if not isinstance(attentionlayer, (Class1, Class2, ...)):
    pooled_features = attn_weights @ features.flatten(start_dim=1)
    assert allclose(pooled_features, output_features)
```

Reply:
Thanks, done